strategies/ReAct.py

```
@@ -56,12 +56,10 @@ class AgentPromptEntity(BaseModel):
 
 
 class ReActAgentStrategy(FilterHistoryMessageByModelFeaturesMixin, AgentStrategy):
-    def __init__(self, runtime, session):
-        super().__init__(runtime, session)
-        self.query = ""
-        self.instruction = ""
-        self.history_prompt_messages = []
-        self.prompt_messages_tools = []
+    query = ""
+    instruction = ""
+    history_prompt_messages = []
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)